static/sass/patterns/_text-blocks.scss

```diff
@@ -0,0 +1,53 @@
+// Text Blocks
+// ===========
+
+.text-block {
+  @include contentblock;
+}
+
+.textblock-headline {
+  @include contentblock-highlight('normal');
+}
+
+.textblock-content {
+  @include contentblock-copy;
+
+  p {
+    &:first-child {
+      margin-top: 0;
+    }
+  }
+}
+
+.headline-contrast {
```

Comment:
Maybe not urgent, but I wonder if we can do something more with this small-screen layout. Centered feels like it's not using the space. Even just bumping up the small-screen font size might help as a quick solution?

Reply:
It felt weird to me too, but left aligned was worse with our line breaks. I can play with the sizes and maybe remove or faux remove the line breaks (or faux add them at wider sizes). I can take some time to see if something else flows better.